pkgs/by-name/ms/mslicer/package.nix

```
@@ -0,0 +1,56 @@
+{
+  fetchFromGitHub,
+  lib,
+  libglvnd,
+  libxkbcommon,
+  nix-update-script,
+  rustPlatform,
+  vulkan-loader,
+  wayland,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mslicer";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "connorslade";
+    repo = pname;
```

```suggestion
    repo = "mslicer";
```

See https://github.com/NixOS/nixpkgs/issues/277994#issuecomment-1876291525.

---

oops! fixed; thanks for catching